step1-01/html-demo/README.md

```diff
@@ -1,6 +1,6 @@
 ## HTML Demo
 
-The [HTML demo page](http://localhost:8080/step1-01/html-demo/html-demo.html) is a large collection of HTML elements that you will come across during development. The full list of elements can be found on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element).
+The [HTML demo page](https://microsoft.github.io/frontend-bootcamp/step1-01/html-demo/html-demo.html) is a large collection of HTML elements that you will come across during development. The full list of elements can be found on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element).
```

Comment:
I think this should be relative link

Reply:
hmm..i'm okay either way. in this case we aren't going to be working inside of that file at all. just viewing